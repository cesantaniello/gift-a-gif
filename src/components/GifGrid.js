import React from 'react'

export const GifGrid = ({category}) => {

    const getGifs = async() => {

        const url = 'https://api.giphy.com/v1/gifs/search?q=batman&limit=10&api_key=AnCjC7Tgzj8tJO9Jj91ImRnw3mdlMsqp'
        const resp = await fetch(url);
        const {data} = await resp.json();



        console.log(data);
    }

    getGifs();

    return (
        <div>
            <h3>{category}</h3>
        </div>
    )
}
