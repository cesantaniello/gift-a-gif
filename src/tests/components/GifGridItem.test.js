import React from 'react';
import { shallow } from 'enzyme';
import { GifGridItem } from '../../components/GifGridItem';

describe('Pruebas en <GifGridItem />', () => {
    test('Debe de mostrarse correctamente', () => {
        const wrapper = shallow(<GifGridItem  />)
        expect(wrapper).toMatchSnapshot();
    })
})