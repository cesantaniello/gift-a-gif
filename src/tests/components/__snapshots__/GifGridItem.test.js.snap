// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> Debe de mostrarse correctamente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
